.gitignore

```diff
@@ -5,3 +5,4 @@ feeds
 .env
 .envrc
 .direnv
+notifications-inventory.yml
```

Comment:
It seems like we are undergoing a shift here (or maybe we already did in past PRs and I didn't realize it) where we are storing secrets in the inventory.yml so it can't be committed to this public repo. Where are the production inventory files being stored currently? I know we have talked about secret management a couple times but I don't remember what our conclusions were.

Reply:
We've been committing example inventory files that match an inventory file you could run, but with correctly set variables.  I've been treating it similar to having a committed .env.template that you copy over when making the actual .env.   This is more confusing than useful though, I realize, and will set up a better way.